BnbnavNetClient/Services/NetworkOperations/NodeMoveOperation.cs

```diff
@@ -53,11 +53,7 @@ public override void Render(MapView mapView, DrawingContext context)
         var movingRect = _toUpdate.BoundingRect(mapView);
         var movedRect = _updateTo.BoundingRect(mapView);
 
-        ExtendedLine lineBetween = new()
-        {
-            Point1 = movingRect.Center,
-            Point2 = movedRect.Center
-        };
+        ExtendedLine lineBetween = new(movingRect.Center, movedRect.Center);
```

Comment:
ask vrabb about this, but IMO it's better to var lineBetween = new ExtendedLine(. . .)